### PR TITLE
Format after #1988 and #1968

### PR DESCRIPTION
**Pull request by sazzad16**
-  Fix #1988 
-  Modify #1988 according to #1968 
- Format and re-order BuilderFactory

### CODE REVIEW

**Comment by gkorland** [comment]
@sazzad16 it is really hard to see what is #2078 is adding on top of  #1968

**Comment by sazzad16** [comment]
@gkorland 

1. Almost all of `Jedis` methods that takes a `count` param, receives it as an `int`. There are only two exceptions in Jedis class. As `zpopmin` is yet to release, I changed its `count` to int.

2. ZPOPMIN without count returns only one Tuple (or none). Instead of returning a `Set<Tuple>` with only one element, it'd be better to return `Tuple`. ZPOPMAX #1968 is changed to be like that. I included this change in this PR.

**Comment by sazzad16** [comment]
@gkorland I forgot that I had changed the same changes in https://github.com/xetorthio/jedis/pull/1968/commits/915a8106e722679548f83a925b168567d891ba51 which are merged through #1968 just now.

**Comment by gkorland** [comment]
So can we close this PR?

**Comment by gkorland** [comment]
or at lease undo all the none relevant changes?

File: src/main/java/redis/clients/jedis/PipelineBase.java [hunk 1]
```diff
@@ -1337,13 +1337,13 @@
   }
 
   @Override
-  public Response<Set<Tuple>> zpopmin(final byte[] key, final int count) {
+  public Response<Set<Tuple>> zpopmin(final String key, final int count) {
```
Comment (gkorland) [inline src/main/java/redis/clients/jedis/PipelineBase.java]: undo?

File: src/main/java/redis/clients/jedis/PipelineBase.java [hunk 2]
```diff
@@ -1337,13 +1337,13 @@
...
     getClient(key).zpopmin(key, count);
     return getResponse(BuilderFactory.TUPLE_ZSET);
   }
 
   @Override
-  public Response<Set<Tuple>> zpopmin(final String key, final int count) {
+  public Response<Set<Tuple>> zpopmin(final byte[] key, final int count) {
```
Comment (gkorland) [inline src/main/java/redis/clients/jedis/PipelineBase.java]: undo?

File: src/main/java/redis/clients/jedis/BuilderFactory.java [hunk 3]
```diff
@@ -13,6 +13,7 @@
 import redis.clients.jedis.util.SafeEncoder;
 
 public final class BuilderFactory {
+
```
Comment (gkorland) [inline src/main/java/redis/clients/jedis/BuilderFactory.java]: undo?

File: src/main/java/redis/clients/jedis/BuilderFactory.java [hunk 4]
```diff
@@ -32,6 +33,7 @@ public String toString() {
       return "double";
     }
   };
+
```
Comment (gkorland) [inline src/main/java/redis/clients/jedis/BuilderFactory.java]: undo?

File: src/main/java/redis/clients/jedis/BuilderFactory.java [hunk 5]
```diff
@@ -43,10 +45,11 @@ public String toString() {
       return "boolean";
     }
   };
+
```
Comment (gkorland) [inline src/main/java/redis/clients/jedis/BuilderFactory.java]: undo?

**Comment by sazzad16** [comment]
@gkorland 
> please undo uneeded changes

done!

**Comment by gkorland** [comment]
@sazzad16 is still seems like a lot of code just moved around

**Pull request closed without merging**